src/App.vue

```diff
@@ -31,7 +31,6 @@ export default {
                 (this as any).$t('global.new_app_version'),
             );
         } else if (userIsNew) {
-            localStorage.clear();
             localStorage.setItem('appVersion', currentVersion);
```

Comment:
@ezra-sg, is it safe to just remove the line for clearing the local storage? or am I breaking something here?